cmd/monaco/integrationtest/v2/test-resources/slo-v2/project/config.yaml

```diff
@@ -0,0 +1,16 @@
+configs:
+- id: custom-sli
+  type: slo-v2
+  config:
+    template: './custom-sli.json'
+    name: custom-sli
+- id: sli-reference
+  type: slo-v2
+  config:
+    name: sli-reference
+    template: './sli-reference.json'
+    parameters:
+      extractedIDs:
+        type: value
+        value:
+          id_HOST_DED360997F66F0C2: HOST-DED360997F66F0C2
```

Comment:
Please don't use real ME-ids. And since it's not important for the test, I would even just let it stay in the json template